build.gradle

```diff
@@ -93,6 +102,15 @@ dependencies {
     }
     api 'com.google.guava:guava:27.1-jre'
 
+    // need at least this commit (or later): https://github.com/bitcoinj/bitcoinj/commit/7629677103dbdd80f14c63b066ed27e1fad47014
+    // in order to work with BouncyCastle 1.64 or later
+    // see issue: https://github.com/bitcoinj/bitcoinj/issues/1951
+    implementation('com.github.bitcoinj:bitcoinj:master-SNAPSHOT') {
```

Comment:
Would prefer if we used a release from the maven repo, which is more reliable than jitpack, like here: https://mvnrepository.com/artifact/org.bitcoinj/bitcoinj-core/0.15.8

Otherwise, if there really is a good reason for using jitpack, we should use a tag, rather than a branch snapshot.

Reply:
@clstrfsck 15.8 is broken with BC, we need a BitcoinJ version containing [this commit](https://github.com/bitcoinj/bitcoinj/commit/7629677103dbdd80f14c63b066ed27e1fad47014), as per my comment in the gradle file, I downloaded 15.8 and it did not contain this fix.

So what to do? Should I target a specific commit? Can I do that with Maven? I thought it required Jitpack?

Reply:
We could target a specific commit for now.  Just use the commit number instead of branch-SNAPSHOT.

Reply:
Fixed